src/main.cpp

```diff
@@ -26,7 +26,9 @@ int main(int argc, char *argv[])
     std::string driveLetter = kiwix::getExecutablePath().substr(0, 3);
     UINT driveType = GetDriveTypeA(driveLetter.c_str());
     if(driveType == DRIVE_REMOTE) {
-        int msgboxID = MessageBoxA(NULL, gt("disable-sandbox"), gt("about-kiwix-desktop-title"), MB_YESNO | MB_ICONQUESTION);
+        const char* message = gt("disable-sandbox").toStdString().c_str();
+        const char* title = gt("about-kiwix-desktop-title").toStdString().c_str();
+        int msgboxID = MessageBoxA(NULL, message, title, MB_YESNO | MB_ICONQUESTION);
```

Comment:
How does this behave with translation using alphabet outside of ascii ?
We may use the wide version as `MessageBoxExW`.

Reply:
Good point. i will make that change.